### ISSUE TITLE
CDAField's member validations is of wrong type.

### ISSUE THREAD

**Issue by 0x7f** [contentful/contentful.java#124]
Right now, the class `CDAField` has a member [`Map<String, Object> validations;`](https://github.com/contentful/contentful.java/blob/353a3897ffe570bcc7c7b53030449b5e18485895/src/main/java/com/contentful/java/cda/CDAField.java#L24).

But when having a look at the JSON preview of a content model in the web frontend, I see validations like these:

```
      "validations": [
        {
          "in": [
            "StringA",
            "StringB",
            "StringC"
          ]
        }
      ],
```

or

```
      "validations": [
        {
          "linkContentType": [
            "location"
          ]
        }
      ],
```

So an array of maps of string to some object. When trying to deserialise them, i get the following exception:

```
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_ARRAY but was BEGIN_OBJECT
```

Unfortunately, the [documentation](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/content-types/content-type) is not very clear about whether its a list or just the map. But at least [this example in the documentation](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/editor-interface) is also using an array.

That's why I think the `validations` member is wrong and should be an array of maps. Or am I missing something here?

**Comment by mariobodemann** [comment]
Hey,

thank you for this report. I'll take a look and evaluate on what the best course of action would be.

Greetings,
Mario

**Comment by mariobodemann** [comment]
fixed by #127 

Thank you for reporting and fixing the issue!
